astropy/io/misc/yaml.py

```diff
@@ -222,6 +223,19 @@ def ignore_aliases(self, data):
 AstropyDumper.add_representer(Time, _time_representer)
 AstropyDumper.add_representer(TimeDelta, _timedelta_representer)
 AstropyDumper.add_representer(coords.SkyCoord, _skycoord_representer)
+AstropyDumper.add_representer(np.int32, \
```

Comment:
Don't use the trailing `\`.  This is basically not allowed in new astropy code, and is not needed at anyway here.